docs/installation_guide.md

```diff
@@ -288,6 +311,9 @@ hostname = 192.168.0.2:8888
 4.Re-deploy Harbor refering to previous section "Managing Harbor's lifecycle". 
 
 
+## Performance tuning
+By default, Harbor will limit the CPU quota used by Clair container to 150000 to avoid using up all the CPU resources,it is defined in the docker-compose.clair.yml file. You can modify it according to your hardware resource.
```

Comment:
By default, Harbor will limit the CPU usage by Clair container to 150000 to avoid using up all the CPU resources. This is defined in the `docker-compose.clair.yml` file. You can modify it based on your hardware configuration.